src/staging/src/components/header.js

```diff
@@ -1,12 +1,14 @@
 import React from "react"
 
+import SkipToContent from "./skipToContent"
 import BetaBanner from "../components/betaBanner"
 import MegaMenu from "./megaMenu.js"
 
 export default function Header({ content = [] }) {
 
   return (
     <>
+      <SkipToContent></SkipToContent>
```

Comment:
same as <SkipToContent></SkipToContent>

we can do in React  <SkipToContent /> 
same thing, just for info

Reply:
Hah, that's what I thought, but I figured I'd be consistent with the next line `<BetaBanner></BetaBanner>`.

I'll merge this, but feel free to change later.